scripts/transpilation/transpiled-build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+#
+# Backpack - Skyscanner's Design System
+#
+# Copyright 2016 Skyscanner Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+lerna clean --yes
+
+rm -rf ./dist
+mkdir "dist"
+cd ./packages
+
+for d in */
+do
+  cd "$d" || return
+  if [ "$d" == "bpk-component-barchart/" ]
+  then
+      cp -R "../../node_modules/d3-array" "./d3-array"
+      cp -R "../../node_modules/d3-color" "./d3-color"
+      cp -R "../../node_modules/d3-format" "./d3-format"
+      cp -R "../../node_modules/d3-interpolate" "./d3-interpolate"
+      cp -R "../../node_modules/d3-scale" "./d3-scale"
+      cp -R "../../node_modules/d3-time" "./d3-time"
+      cp -R "../../node_modules/d3-time-format" "./d3-time-format"
+      cp -R "../../node_modules/internmap" "./internmap"
+  fi
+
+  if [ "$d" == "bpk-component-flare/" ] || [ "$d" == "bpk-component-icon/" ]
+  then
+    npm run build
+  fi
```

Comment:
could you add an explanation explaining why these components need a special usecase

Reply:
They have their own build scripts and guplfile. This allows to execute them before transpilation. It's mostly about copying some icons from node_modules. To my mind it's redundant for transpiled build as we have svg import plugin, but I left it here for now in order not to break anything

Reply:
We probably don't need to have it as part of this script as part of general running, setup and releasing we have this separately or you can even in the package.json file before running the script have it run our usual `npm run build`

I'd be wary about doing it here separately incase we change things, this will introduce a new level of change

Reply:
>To my mind it's redundant for transpiled build as we have svg import plugin, but I left it here for now in order not to break anything

We still need it even with a new plugin as we do some transformation of them before we use them to do some specific web thing for the icons which the plugin won't cover